package.json

```diff
@@ -49,11 +49,12 @@
   ],
   "private": true,
   "devDependencies": {
-    "gulp-util": "~2.2.14",
     "gulp": "~3.8.7",
-    "gulp-jshint": "~1.9.2",
-    "gulp-jscs": "~1.3.1",
+    "gulp-jshint": "git://github.com/spalger/gulp-jshint#c18df3a11",
```

Comment:
It's actually just a single commit on top of the main repository, but we can let this stale until this -or a equivalent feature- is released.